string uuid
string labelGroup
---
mesh_msgs/MeshFaceCluster[] clusters
